### ISSUE TITLE
Image cache restriction and how user could make us break 500px Terms of Use

### ISSUE THREAD

**Issue by marsh12th** [500px/api-documentation#204]
Hey, it's me again about image caching rules.
Now that I know that no app can't cache or store any image given by both the 500px site and API, I have a certain situation I want to address.

Let's say I make an App which downloads a few images from your API for whatever reason. The App stores those images in a temporary folder ( hidden / protected or whatever ). When App exits those images get deleted from the user's system.

Fair enough.
But what if...

1. User kills App process in Task Manager. That won't trigger 'Exit' event on App, so images will remain there for undefined time.

2. User just finds a way of extracting those images and saving them.

In both cases, even though App is respecting 500px Terms, the user is using it in a way that let him avoid those laws.

**TL;DR** People can just screenshot images from your site, for instance.

In those cases, who is to blame?
The user for making a bad use of the app, or the app itself for... not being more restrictive? I guess.

Regards

**Comment by mck182** [comment]
Hey,

thanks for checking in. I don't think you need to worry what your users do as long as you don't give them a tool that purposefully breaks/allows to break the terms. In your specific cases, you should not give users a direct access to your temporary cache, but beyond that you're fine :)

Let me know if you have any other questions! I'll close this ticket but feel free to post more comments below.

Best wishes,
Martin, 500px

**Comment by marsh12th** [comment]
Actually I do have one more :P

You state that we should contact sales@500px for non-watermarked images... But I'm getting no response haha so can you tell how does that work?

**Comment by mck182** [comment]
Hey,

we've checked and we haven't received any email asking about that.

Can you please resend your email? We'll get back to you.

Best wishes,
Martin, 500px

**Comment by marsh12th** [comment]
Resent. Hope you recieve it.

Regards
